src/plays/typing-speed-test/styles.css

```diff
@@ -0,0 +1,14 @@
+/* enter stlyes here */
+.result__modal {

```

Comment:
Can we scope this style name to the play.. with a prefix?

Reply:
Sure i'll fix it ...!